src/client/src/pages/NewProjectPage/messagesReactNative.ts

```diff
@@ -0,0 +1,17 @@
+import { defineMessages } from "react-intl";
+
+const messagesReactNative = defineMessages({
+  header: {
+    id: "newProjectPage.reactNative.header",
+    defaultMessage: "Create Your React Native App in Seconds (WIP)",
+  },
+  body: {
+    id: "newProjectPage.reactNative.body",
+    defaultMessage: "Give your react native project a name, choose where to create it, then click 'Next' to get started.",
```

Comment:
We shiould correct "React Native" casing or remove it from the body text.
```suggestion
    defaultMessage: "Give your project a name, choose where to create it, then click 'Next' to get started.",
```